compiler/rustc_resolve/src/diagnostics.rs

```diff
@@ -3026,14 +3024,3 @@ fn is_span_suitable_for_use_injection(s: Span) -> bool {
     // import or other generated ones
     !s.from_expansion()
 }
-
-/// Convert the given number into the corresponding ordinal
-pub(crate) fn ordinalize(v: usize) -> String {
```

Comment:
this conflicts with #125042, but I think that one can use just the number too

Reply:
In https://github.com/rust-lang/rust/pull/125042, the function is copied, not referenced (put aside whether it is good or not).